app/src/pages/settings/ModelsTable.tsx

```diff
@@ -229,13 +217,98 @@ export function ModelsTable({
         accessorKey: "namePattern",
         cell: TextCell,
       },
-      makeCostColumn("input", "input cost"),
-      makeCostColumn("output", "output cost"),
-      makeCostColumn("cacheRead", "cache read cost"),
-      makeCostColumn("cacheWrite", "cache write cost"),
-      makeCostColumn("promptAudio", "prompt audio cost"),
-      makeCostColumn("completionAudio", "completion audio cost"),
-      makeCostColumn("reasoning", "reasoning cost"),
+      {
+        header: "prompt cost",
+        id: "input",
+        accessorFn: (row) => getRowCost(row, "input"),
+        meta: { textAlign: "right" },
+        sortingFn: sortCostColumnFn,
+        sortUndefined: "last",
```

Comment:
I don't think this matters anymore if the accessorFn is always going to return a number (since input and output are required)

Reply:
sorta, yeah. I think in most cases you're just trying to see what the overall price of the models are.